admin-dev/themes/new-theme/js/product-page/product-bulk-combinations.js

```diff
@@ -171,17 +176,25 @@ export default function() {
       globalProductSubmitButton.submit();
     },
     'syncToPricingTab': function syncToPricingTab() {
-      var newPrice = finalPrice.val();
       $('tr.combination').toArray().forEach((item) => {
-        var jQueryRow = $('#'+item.id);
-        var jQueryFinalPriceEl = jQueryRow.find(finalPriceSelector);
-        var impactOnPriceEl = jQueryRow.find(impactOnPriceSelector);
-        var impactOnPrice = impactOnPriceEl.val();
-
-        jQueryFinalPriceEl.data('price', newPrice);
-        // calculate new price
-        var newFinalPrice = new Number(newPrice) + new Number(impactOnPrice);
-        jQueryFinalPriceEl.text(ps_round(newFinalPrice, 6));
+        var tableRow = $('#'+item.id);
+        // We need this because there is a specific data="smthg" attribute so we can't use data() function
+        var attributeId = tableRow.attr('data');
+
+        // Get combination final price value from combination form
+        var finalPrice = priceCalculation.getCombinationFinalPriceTaxExcludedById(attributeId);
+        var finalPriceLabel = tableRow.find('.attribute-finalprice span.final-price');
+        finalPriceLabel.html(finalPrice);
+
+        // Update ecotax preview (tax included)
+        var combinationEcotaxTI = priceCalculation.getCombinationEcotaxTaxIncludedById(attributeId);
+        if (combinationEcotaxTI === 0) {
+          combinationEcotaxTI = priceCalculation.getProductEcotaxTaxIncluded();
+        }
+        var ecoTaxLabel = tableRow.find('.attribute-finalprice span.attribute-ecotax');
+        ecoTaxLabel.html(Number(ps_round(combinationEcotaxTI, 2)).toFixed(2)); // 2 digits for short
```

Comment:
Same here

Reply:
Same reason as above, in the list the space is limited which is why I chose to only display two digits